app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -198,7 +200,7 @@ ledgerRecoveryFailedTitle=Recovery Failed
 ledgerRecoveryFailedMessage=Please re-enter keys or try different keys.
 ledgerRecoveryNetworkFailedTitle=Network Error
 ledgerRecoveryNetworkFailedMessage=Please check your internet connection and try again.
-ledgerRecoveryContent=The balance of the recovered wallet will be transferred to your new Brave wallet. The old wallet will still exist as an empty wallet.
+ledgerRecoveryContent=Your previous wallet will now be used. Your new wallet will be discarded.
```

Comment:
Just to confirm:
@mrose17 when you do a restore, it'll use the previous wallet (what you just restored), right? So the old text here (about transferring the balance to your new wallet) was wrong?